src/types/media-session.ts

```diff
@@ -31,9 +31,14 @@ export class GenesysCloudMediaSession extends MediaSession {
   }
 
   onIceStateChange () {
-    const state = this.pc.iceConnectionState;
+    const iceState = this.pc.iceConnectionState;
+    const connectionState = this.pc.connectionState;
```

Comment:
So this is related to the iceConnectionState but isn't exactly the same thing. I would say we should have two separate handlers/logs for them. It looks like stanza doesn't expose the connectionState handler like they do for the ice state so we probably just need to add one in the constructor. Something like this:
```
this.pc.addEventListener('connectionState', this.onConnectionStateChange.bind(this))
```
and we'll remove the connectionState stuff from this fn. Does that make sense?

Reply:
@jensengar Just to verify - you think we should leave the the iceConnectionState how it is and create a handler for connectionState?

Reply:
@zservies not quite. I like what you have here for the iceConnectionState handler stuff, just remove the stuff relating to `pc.connectionState`. Then create a `connectionStateChange` function that basically does the same thing, but for the `pc.connectionState` rather than the `pc.iceConnectionState`.

Reply:
@jensengar Yeah thats what I was essentially trying to get at.